test/views/ShutMucrConfirmationViewSpec.scala

```diff
@@ -17,75 +17,54 @@
+  private val view: Html = shutMucrConformationPage()(request, Flash(Map(FlashKeys.MUCR -> exampleMucr)), messages)
 
   "Shut Mucr Confirmation View" should {
 
     "have proper labels for messages" in {
 
-      assertMessage(title, "MUCR shut")
-      assertMessage(confirmationInfo, "The reference of the MUCR shut:")
-      assertMessage(additionalNote, "You might want to take a screenshot of this for your records.")
-    }
-
-    "display same page title as header" in {
-
-      val view = createView()
-      view.title() must include(view.getElementsByTag("h1").text())
-    }
+      val messages = messagesApi.preferred(request)
 
-    "display page heading inside highlight box" in {
-
-      val view = createView()
-
-      getElementById(view, "highlight-box-heading").text() must equal(messages(title))
-      getElementByCss(view, ".govuk-box-highlight").text() must include(messages(title))
+      messages must haveTranslationFor("shutMucr.confirmation.tab.heading")
+      messages must haveTranslationFor("shutMucr.confirmation.heading")
+      messages must haveTranslationFor("shutMucr.confirmation.shutOrDepart")
     }
 
-    "display confirmation information with MUCR inside highlight box" in {
+    "display page reference" in {
 
-      val view = createView()
-
-      getElementById(view, "highlight-box-info").text() must equal(messages(confirmationInfo))
-      getElementByCss(view, ".govuk-box-highlight").text() must include(messages(confirmationInfo))
+      view.getElementById("highlight-box-heading").text() mustBe messages("shutMucr.confirmation.heading")
     }
 
-    "display reference MUCR inside highlight box" in {
-
-      val mucr = "MUCR1234567890"
-      val view = createView(Some(mucr))
+    "have status information" in {
 
-      getElementById(view, "highlight-box-reference").text() must equal(mucr)
-      getElementByCss(view, ".govuk-box-highlight").text() must include(mucr)
+      view.getElementById("status-info").text() mustBe messages("movement.confirmation.statusInfo")
     }
 
-    "display '-' as reference MUCR when no value provided in flash" in {
+    "have what next section" in {
 
-      getElementById(createView(), "highlight-box-reference").text() must equal("-")
+      view.getElementById("what-next").text() mustBe messages("movement.confirmation.whatNext")
     }
 
-    "display additional note" in {
+    "have next steps section" in {
 
-      getElementById(createView(), "additional-note").text() must equal(messages(additionalNote))
+      view.getElementById("next-steps").text() mustBe messages("shutMucr.confirmation.shutOrDepart")
     }
 
-    "display 'Back to start page' button that links to Start Page" in {
+    "display 'Back to start page' button on page" in {
 
-      val button = getElementByCss(createView(), ".button")
-      button.text() must equal(messages(backToStartPageCaption))
-      button.attr("href") must equal("/customs-movements/start")
```

Comment:
Why is it removed? you can use `haveHref(...)` matcher for this.

Reply:
Fixed